libs/ArgumentHandler/ArgumentHandler.hpp

```diff
@@ -41,6 +41,9 @@ namespace argument_handler
 
             /// @brief Port to bind to.
             unsigned short port;
+
+            /// @brief Room configurations.
+            short *configs;
```

Comment:
👀

Reply:
configurations are used to add modifier on room creation. There are raw pointer because they are used by the network.